### PR TITLE
CS: Operator => should not be arounded by multi-line whitespaces.

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

File: tests/ZendTest/Form/Element/CollectionTest.php [hunk 1]
```diff
@@ -373,8 +373,7 @@ public function testExtractFromObjectDoesntTouchOriginalObject()
         $form->bind($product);
 
         $form->setData(
-            array("product" =>
-                array(
+            array("product" => array(
                     "name" => "franz",
```
Comment (Maks3w) [inline tests/ZendTest/Form/Element/CollectionTest.php]: In this case array elements has too much indentation

File: tests/ZendTest/Ldap/AbstractOnlineTestCase.php [hunk 2]
```diff
@@ -93,34 +93,26 @@ protected function createDn($dn)
     protected function prepareLDAPServer()
     {
         $this->nodes = array(
-            $this->createDn('ou=Node,')          =>
-            array("objectClass" => "organizationalUnit",
+            $this->createDn('ou=Node,')          => array("objectClass" => "organizationalUnit",
```
Comment (Maks3w) [inline tests/ZendTest/Ldap/AbstractOnlineTestCase.php]: It's desirable to remove vertical alignment spaces

File: tests/ZendTest/Ldap/AbstractOnlineTestCase.php [hunk 3]
```diff
@@ -93,34 +93,26 @@ protected function createDn($dn)
...
                   "ou"          => "Node",
                   "postalCode"  => "1234"),
```
Comment (Maks3w) [inline tests/ZendTest/Ldap/AbstractOnlineTestCase.php]: For preserve the symmetry closing brace should move to his own line

File: tests/ZendTest/Form/Element/CollectionTest.php (hunk 1 above)
Comment (keradus) [inline tests/ZendTest/Form/Element/CollectionTest.php]: done

File: tests/ZendTest/Ldap/AbstractOnlineTestCase.php (hunk 3 above)
Comment (keradus) [inline tests/ZendTest/Ldap/AbstractOnlineTestCase.php]: done

File: tests/ZendTest/Ldap/AbstractOnlineTestCase.php (hunk 2 above)
Comment (keradus) [inline tests/ZendTest/Ldap/AbstractOnlineTestCase.php]: done

**Comment by keradus** [comment]
why, why the CS on develop branch are broken :(

**Comment by keradus** [comment]
yeah, make a review and after 3 weeks close PR ;)

**Pull request closed without merging**